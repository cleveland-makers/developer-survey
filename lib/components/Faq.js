import React from 'react';
import { Link } from 'react-router-dom';

const Faq = () => {
  return (
    <div>
      <h3>FAQ</h3>
      <span>
        Why are we doing things.
      </span>
    </div>
  )
};

export default Faq;
