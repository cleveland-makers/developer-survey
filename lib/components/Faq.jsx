import React from 'react';
import RaisedButton from 'material-ui/RaisedButton';
import FullWidthSection from './FullWidthSection';

const styles = {
  h1: {
    color: '#730006',
    fontFamily: 'Play, serif',
    fontSize: '50px',
    fontWeight: '600',
    lineHeight: '1.08',
    marginBottom: '40px',
    marginTop: '20px',
    textShadow: '2px 2px 5px #F24932',
    textTransform: 'uppercase',
  },
  div: {
    margin: 'auto',
    width: '60%',
  },
  buttonMain: {
    height: '45px',
    marginRight: '16px',
    width: '110px',
  },
  buttonMainLabel: {
    lineHeight: '45px',
  },
  buttonSecondary: {
    backgroundColor: '#F7F5F4',
    border: '2px solid #4556A5',
    borderRadius: '2px',
    color: '#4556A5',
    padding: '15px 32px',
    textAlign: 'center',
    textDecoration: 'none',
    display: 'inline-block',
    font: '16px Roboto, sans-serif',
    fontWeight: 'bold',
  },
  root: {
    backgroundColor: '#8097ad',
    overflow: 'hidden',
  },
};

const Faq = () => (
  <FullWidthSection style={styles.root}>
    <section style={styles.div}>
      <div>
        <h1 style={styles.h1}>Welcome to the CLE Tech Survey</h1>
        <p>
          Popular tech surveys donâ€™t seem to get Cleveland. So, we made our own
          Cleveland-based survey to try to get a more accurate idea of what goes on
          in Cleveland tech.
        </p>
        <p>
          The purpose of this survey is to collect and present information about
          salary, education, and experience for those in Cleveland Technology to
          get an understanding of the influence different components have on
          salary calculations.
        </p>
        <p>
          The information collected is anonymous and will be accessible to all, in
          downloadable (INSERT FILE FORMAT TYPE HERE) format, once the survey
          collection period is over (INSERT DATE OF COMPLETION HERE).
        </p>
        <p>
          The survey takes about 7 minutes to complete.
        </p>
      </div>
      <div>
        <RaisedButton
          backgroundColor="#730006"
          href="/survey"
          label="Start"
          labelColor="#F7F5F4"
          labelStyle={styles.buttonMainLabel}
          style={styles.buttonMain}
        />
      </div>
    </section>
  </FullWidthSection>
);

export default Faq;
