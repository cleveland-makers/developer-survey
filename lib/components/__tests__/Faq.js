import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Faq from '../Faq';

configure({ adapter: new Adapter() });

describe('Faq', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Faq />);

    expect(wrapper.find('h1').length).toBe(1);
    expect(wrapper).toMatchSnapshot();
  });
});
