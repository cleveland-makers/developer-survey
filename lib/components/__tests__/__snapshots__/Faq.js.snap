// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Faq renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Faq />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <section
        style={
          Object {
            "margin": "auto",
            "width": "60%",
          }
        }
      >
        <div>
          <h1
            style={
              Object {
                "color": "#730006",
                "fontFamily": "Play, serif",
                "fontSize": "50px",
                "fontWeight": "600",
                "lineHeight": "1.08",
                "marginBottom": "40px",
                "marginTop": "20px",
                "textShadow": "2px 2px 5px #F24932",
                "textTransform": "uppercase",
              }
            }
          >
            Welcome to the CLE Tech Survey
          </h1>
          <p>
            Popular tech surveys don’t seem to get Cleveland. So, we made our own Cleveland-based survey to try to get a more accurate idea of what goes on in Cleveland tech.
          </p>
          <p>
            The purpose of this survey is to collect and present information about salary, education, and experience for those in Cleveland Technology to get an understanding of the influence different components have on salary calculations.
          </p>
          <p>
            The information collected is anonymous and will be accessible to all, in downloadable (INSERT FILE FORMAT TYPE HERE) format, once the survey collection period is over (INSERT DATE OF COMPLETION HERE).
          </p>
          <p>
            The survey takes about 7 minutes to complete.
          </p>
        </div>
        <div>
          <RaisedButton
            backgroundColor="#730006"
            disabled={false}
            fullWidth={false}
            href="/survey"
            label="Start"
            labelColor="#F7F5F4"
            labelPosition="after"
            labelStyle={
              Object {
                "lineHeight": "45px",
              }
            }
            primary={false}
            secondary={false}
            style={
              Object {
                "height": "45px",
                "marginRight": "16px",
                "width": "110px",
              }
            }
          />
        </div>
      </section>,
      "style": Object {
        "backgroundColor": "#8097ad",
        "overflow": "hidden",
      },
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div>
            <h1
              style={
                Object {
                  "color": "#730006",
                  "fontFamily": "Play, serif",
                  "fontSize": "50px",
                  "fontWeight": "600",
                  "lineHeight": "1.08",
                  "marginBottom": "40px",
                  "marginTop": "20px",
                  "textShadow": "2px 2px 5px #F24932",
                  "textTransform": "uppercase",
                }
              }
            >
              Welcome to the CLE Tech Survey
            </h1>
            <p>
              Popular tech surveys don’t seem to get Cleveland. So, we made our own Cleveland-based survey to try to get a more accurate idea of what goes on in Cleveland tech.
            </p>
            <p>
              The purpose of this survey is to collect and present information about salary, education, and experience for those in Cleveland Technology to get an understanding of the influence different components have on salary calculations.
            </p>
            <p>
              The information collected is anonymous and will be accessible to all, in downloadable (INSERT FILE FORMAT TYPE HERE) format, once the survey collection period is over (INSERT DATE OF COMPLETION HERE).
            </p>
            <p>
              The survey takes about 7 minutes to complete.
            </p>
          </div>,
          <div>
            <RaisedButton
              backgroundColor="#730006"
              disabled={false}
              fullWidth={false}
              href="/survey"
              label="Start"
              labelColor="#F7F5F4"
              labelPosition="after"
              labelStyle={
                Object {
                  "lineHeight": "45px",
                }
              }
              primary={false}
              secondary={false}
              style={
                Object {
                  "height": "45px",
                  "marginRight": "16px",
                  "width": "110px",
                }
              }
            />
          </div>,
        ],
        "style": Object {
          "margin": "auto",
          "width": "60%",
        },
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h1
                style={
                  Object {
                    "color": "#730006",
                    "fontFamily": "Play, serif",
                    "fontSize": "50px",
                    "fontWeight": "600",
                    "lineHeight": "1.08",
                    "marginBottom": "40px",
                    "marginTop": "20px",
                    "textShadow": "2px 2px 5px #F24932",
                    "textTransform": "uppercase",
                  }
                }
              >
                Welcome to the CLE Tech Survey
              </h1>,
              <p>
                Popular tech surveys don’t seem to get Cleveland. So, we made our own Cleveland-based survey to try to get a more accurate idea of what goes on in Cleveland tech.
              </p>,
              <p>
                The purpose of this survey is to collect and present information about salary, education, and experience for those in Cleveland Technology to get an understanding of the influence different components have on salary calculations.
              </p>,
              <p>
                The information collected is anonymous and will be accessible to all, in downloadable (INSERT FILE FORMAT TYPE HERE) format, once the survey collection period is over (INSERT DATE OF COMPLETION HERE).
              </p>,
              <p>
                The survey takes about 7 minutes to complete.
              </p>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Welcome to the CLE Tech Survey",
                "style": Object {
                  "color": "#730006",
                  "fontFamily": "Play, serif",
                  "fontSize": "50px",
                  "fontWeight": "600",
                  "lineHeight": "1.08",
                  "marginBottom": "40px",
                  "marginTop": "20px",
                  "textShadow": "2px 2px 5px #F24932",
                  "textTransform": "uppercase",
                },
              },
              "ref": null,
              "rendered": "Welcome to the CLE Tech Survey",
              "type": "h1",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Popular tech surveys don’t seem to get Cleveland. So, we made our own Cleveland-based survey to try to get a more accurate idea of what goes on in Cleveland tech.",
              },
              "ref": null,
              "rendered": "Popular tech surveys don’t seem to get Cleveland. So, we made our own Cleveland-based survey to try to get a more accurate idea of what goes on in Cleveland tech.",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "The purpose of this survey is to collect and present information about salary, education, and experience for those in Cleveland Technology to get an understanding of the influence different components have on salary calculations.",
              },
              "ref": null,
              "rendered": "The purpose of this survey is to collect and present information about salary, education, and experience for those in Cleveland Technology to get an understanding of the influence different components have on salary calculations.",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "The information collected is anonymous and will be accessible to all, in downloadable (INSERT FILE FORMAT TYPE HERE) format, once the survey collection period is over (INSERT DATE OF COMPLETION HERE).",
              },
              "ref": null,
              "rendered": "The information collected is anonymous and will be accessible to all, in downloadable (INSERT FILE FORMAT TYPE HERE) format, once the survey collection period is over (INSERT DATE OF COMPLETION HERE).",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "The survey takes about 7 minutes to complete.",
              },
              "ref": null,
              "rendered": "The survey takes about 7 minutes to complete.",
              "type": "p",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <RaisedButton
              backgroundColor="#730006"
              disabled={false}
              fullWidth={false}
              href="/survey"
              label="Start"
              labelColor="#F7F5F4"
              labelPosition="after"
              labelStyle={
                Object {
                  "lineHeight": "45px",
                }
              }
              primary={false}
              secondary={false}
              style={
                Object {
                  "height": "45px",
                  "marginRight": "16px",
                  "width": "110px",
                }
              }
            />,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "backgroundColor": "#730006",
              "disabled": false,
              "fullWidth": false,
              "href": "/survey",
              "label": "Start",
              "labelColor": "#F7F5F4",
              "labelPosition": "after",
              "labelStyle": Object {
                "lineHeight": "45px",
              },
              "primary": false,
              "secondary": false,
              "style": Object {
                "height": "45px",
                "marginRight": "16px",
                "width": "110px",
              },
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": "div",
        },
      ],
      "type": "section",
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <section
          style={
            Object {
              "margin": "auto",
              "width": "60%",
            }
          }
        >
          <div>
            <h1
              style={
                Object {
                  "color": "#730006",
                  "fontFamily": "Play, serif",
                  "fontSize": "50px",
                  "fontWeight": "600",
                  "lineHeight": "1.08",
                  "marginBottom": "40px",
                  "marginTop": "20px",
                  "textShadow": "2px 2px 5px #F24932",
                  "textTransform": "uppercase",
                }
              }
            >
              Welcome to the CLE Tech Survey
            </h1>
            <p>
              Popular tech surveys don’t seem to get Cleveland. So, we made our own Cleveland-based survey to try to get a more accurate idea of what goes on in Cleveland tech.
            </p>
            <p>
              The purpose of this survey is to collect and present information about salary, education, and experience for those in Cleveland Technology to get an understanding of the influence different components have on salary calculations.
            </p>
            <p>
              The information collected is anonymous and will be accessible to all, in downloadable (INSERT FILE FORMAT TYPE HERE) format, once the survey collection period is over (INSERT DATE OF COMPLETION HERE).
            </p>
            <p>
              The survey takes about 7 minutes to complete.
            </p>
          </div>
          <div>
            <RaisedButton
              backgroundColor="#730006"
              disabled={false}
              fullWidth={false}
              href="/survey"
              label="Start"
              labelColor="#F7F5F4"
              labelPosition="after"
              labelStyle={
                Object {
                  "lineHeight": "45px",
                }
              }
              primary={false}
              secondary={false}
              style={
                Object {
                  "height": "45px",
                  "marginRight": "16px",
                  "width": "110px",
                }
              }
            />
          </div>
        </section>,
        "style": Object {
          "backgroundColor": "#8097ad",
          "overflow": "hidden",
        },
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div>
              <h1
                style={
                  Object {
                    "color": "#730006",
                    "fontFamily": "Play, serif",
                    "fontSize": "50px",
                    "fontWeight": "600",
                    "lineHeight": "1.08",
                    "marginBottom": "40px",
                    "marginTop": "20px",
                    "textShadow": "2px 2px 5px #F24932",
                    "textTransform": "uppercase",
                  }
                }
              >
                Welcome to the CLE Tech Survey
              </h1>
              <p>
                Popular tech surveys don’t seem to get Cleveland. So, we made our own Cleveland-based survey to try to get a more accurate idea of what goes on in Cleveland tech.
              </p>
              <p>
                The purpose of this survey is to collect and present information about salary, education, and experience for those in Cleveland Technology to get an understanding of the influence different components have on salary calculations.
              </p>
              <p>
                The information collected is anonymous and will be accessible to all, in downloadable (INSERT FILE FORMAT TYPE HERE) format, once the survey collection period is over (INSERT DATE OF COMPLETION HERE).
              </p>
              <p>
                The survey takes about 7 minutes to complete.
              </p>
            </div>,
            <div>
              <RaisedButton
                backgroundColor="#730006"
                disabled={false}
                fullWidth={false}
                href="/survey"
                label="Start"
                labelColor="#F7F5F4"
                labelPosition="after"
                labelStyle={
                  Object {
                    "lineHeight": "45px",
                  }
                }
                primary={false}
                secondary={false}
                style={
                  Object {
                    "height": "45px",
                    "marginRight": "16px",
                    "width": "110px",
                  }
                }
              />
            </div>,
          ],
          "style": Object {
            "margin": "auto",
            "width": "60%",
          },
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h1
                  style={
                    Object {
                      "color": "#730006",
                      "fontFamily": "Play, serif",
                      "fontSize": "50px",
                      "fontWeight": "600",
                      "lineHeight": "1.08",
                      "marginBottom": "40px",
                      "marginTop": "20px",
                      "textShadow": "2px 2px 5px #F24932",
                      "textTransform": "uppercase",
                    }
                  }
                >
                  Welcome to the CLE Tech Survey
                </h1>,
                <p>
                  Popular tech surveys don’t seem to get Cleveland. So, we made our own Cleveland-based survey to try to get a more accurate idea of what goes on in Cleveland tech.
                </p>,
                <p>
                  The purpose of this survey is to collect and present information about salary, education, and experience for those in Cleveland Technology to get an understanding of the influence different components have on salary calculations.
                </p>,
                <p>
                  The information collected is anonymous and will be accessible to all, in downloadable (INSERT FILE FORMAT TYPE HERE) format, once the survey collection period is over (INSERT DATE OF COMPLETION HERE).
                </p>,
                <p>
                  The survey takes about 7 minutes to complete.
                </p>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Welcome to the CLE Tech Survey",
                  "style": Object {
                    "color": "#730006",
                    "fontFamily": "Play, serif",
                    "fontSize": "50px",
                    "fontWeight": "600",
                    "lineHeight": "1.08",
                    "marginBottom": "40px",
                    "marginTop": "20px",
                    "textShadow": "2px 2px 5px #F24932",
                    "textTransform": "uppercase",
                  },
                },
                "ref": null,
                "rendered": "Welcome to the CLE Tech Survey",
                "type": "h1",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Popular tech surveys don’t seem to get Cleveland. So, we made our own Cleveland-based survey to try to get a more accurate idea of what goes on in Cleveland tech.",
                },
                "ref": null,
                "rendered": "Popular tech surveys don’t seem to get Cleveland. So, we made our own Cleveland-based survey to try to get a more accurate idea of what goes on in Cleveland tech.",
                "type": "p",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "The purpose of this survey is to collect and present information about salary, education, and experience for those in Cleveland Technology to get an understanding of the influence different components have on salary calculations.",
                },
                "ref": null,
                "rendered": "The purpose of this survey is to collect and present information about salary, education, and experience for those in Cleveland Technology to get an understanding of the influence different components have on salary calculations.",
                "type": "p",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "The information collected is anonymous and will be accessible to all, in downloadable (INSERT FILE FORMAT TYPE HERE) format, once the survey collection period is over (INSERT DATE OF COMPLETION HERE).",
                },
                "ref": null,
                "rendered": "The information collected is anonymous and will be accessible to all, in downloadable (INSERT FILE FORMAT TYPE HERE) format, once the survey collection period is over (INSERT DATE OF COMPLETION HERE).",
                "type": "p",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "The survey takes about 7 minutes to complete.",
                },
                "ref": null,
                "rendered": "The survey takes about 7 minutes to complete.",
                "type": "p",
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <RaisedButton
                backgroundColor="#730006"
                disabled={false}
                fullWidth={false}
                href="/survey"
                label="Start"
                labelColor="#F7F5F4"
                labelPosition="after"
                labelStyle={
                  Object {
                    "lineHeight": "45px",
                  }
                }
                primary={false}
                secondary={false}
                style={
                  Object {
                    "height": "45px",
                    "marginRight": "16px",
                    "width": "110px",
                  }
                }
              />,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "backgroundColor": "#730006",
                "disabled": false,
                "fullWidth": false,
                "href": "/survey",
                "label": "Start",
                "labelColor": "#F7F5F4",
                "labelPosition": "after",
                "labelStyle": Object {
                  "lineHeight": "45px",
                },
                "primary": false,
                "secondary": false,
                "style": Object {
                  "height": "45px",
                  "marginRight": "16px",
                  "width": "110px",
                },
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": "div",
          },
        ],
        "type": "section",
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
